src/test/java/core/basesyntax/service/impl/ReportServiceImplTest.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.service.ReportService;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class ReportServiceImplTest {
+    private static Map<String, Integer> testStorage;
+    private static ReportService reportService;
+
+    @BeforeAll
+    static void beforeAll() {
+        reportService = new ReportServiceImpl();
+        testStorage = new HashMap<>();
+    }
+
+    @AfterEach
+    void tearDown() {
+        testStorage.clear();
+    }
+
+    @Test
+    void createReport_storageWithThreeElements_ok() {
+        testStorage.put("apple", 599);
+        testStorage.put("peach", 135);
+        testStorage.put("grape", 1180);
+        String expected = "apple,599"
+                + System.lineSeparator()
+                + "peach,135"
+                + System.lineSeparator()
+                + "grape,1180";
```

Comment:
You could try use here and similar places: List<String> expected  = List.of("apple,599", "peach,135", "grape,1180")

Reply:
I can't, cos type of my report is String